levels/inside_perimeter/objectives/api-02-async-await/example.js

```diff
@@ -0,0 +1,10 @@
+const { readdir } = require("fs").promises;
+
+async function findLastFileInDir(dirPath) {
+  // TODO: return the last file in dirPath!
+}
+
+(async function () {
```

Comment:
Not sure how we feel bout this way of writing a "test run" of the user's function, but I had a hard time thinking of another solution that wouldn't end up being a time sink